### PR TITLE
Windows support

### PR DESCRIPTION
**Pull request by MekDrop**
I think with this node and yarn could be installed on windows without problems too!

### CODE REVIEW

File: src/BinLinker.php [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+    /**
+     * Links bin item
+     *
+     * @param string $from Path from where to create symlink
+     * @param string $to Path to where create symlink
+     */
+    public function linkBin($from, $to)
+    {
+        if ($this->osType === 'win') {
+            $this->filesystem->dumpFile(
```
Comment (mariusbuescher) [inline src/BinLinker.php:60]: This is not really a link. Isn't there another way of generating a link on windows?

**Comment by mariusbuescher** [comment]
Could you also please rebase or merge master?

File: src/BinLinker.php (hunk 1 above)
Comment (MekDrop) [reply src/BinLinker.php:60]: Sadly, on windows symlink creation for some reason works only when command is running with administrator rights. And that is not very common between servers.

**Comment by MekDrop** [comment]
I just updated this pull request.

File: src/BinLinker.php (hunk 1 above)
Comment (mariusbuescher) [reply src/BinLinker.php:60]: Mmmh, OK. I think the last version of Windows I really used was XP, so I have no idea what is going on on these machines.

**Pull request merged**